lib/simple-dom/document.js

```diff
@@ -145,22 +145,32 @@ if(Object.defineProperty) {
 	});
 }
 
-function addMissingMethod(name){
-	if (Document.prototype[name])return
+function warnMissingMethod(name, methodOrNot = true){
 	var hasWarned = false;
-	Object.defineProperty(Document.prototype, name, {
-		get: function(){
-			if(!hasWarned) {
+	var warning = "document."+name+" is not implemented in can-simple-dom with witch SSR occurs.  Please make sure this code runs after the document has finished loading, with guide at bitballs (turning off SSR) under donejs.com."
+	if (methodOrNot){
```

Comment:
I'm not sure what this is about.. I think it's never used, right? I think we can just do this on the getter, always.